modules/ipi-install-firmware-requirements-for-installing-with-virtual-media.adoc

```diff
@@ -6,32 +6,41 @@
 [id='ipi-install-firmware-requirements-for-installing-with-virtual-media_{context}']
 = Firmware requirements for installing with virtual media
 
-The installer for installer-provisioned {product-title} clusters validates the hardware and firmware compatibility with Redfish virtual media. The following table lists the minimum firmware versions tested and verified to work for installer-provisioned {product-title} clusters deployed by using Redfish virtual media.
+The installation program for installer-provisioned {product-title} clusters validates the hardware and firmware compatibility with Redfish virtual media. The installation program will not begin installation on a node if the node firmware is not compatible. The following tables list the minimum firmware versions tested and verified to work for installer-provisioned {product-title} clusters deployed by using Redfish virtual media.
 
-.Firmware compatibility for Redfish virtual media
+[NOTE]
+====
+Red Hat does not test every combination of firmware, hardware, or other third-party components. For further information about third-party support, see link:https://access.redhat.com/third-party-software-support[Red Hat third-party support policy]. See the hardware documentation for the nodes or contact the hardware vendor for information about updating the firmware.
+====
+
+.Firmware compatibility for HP hardware with Redfish virtual media 
 [frame="topbot", options="header"]
+[cols="1,1,1"]
 |====
-|Hardware| Model | Management | Firmware versions
-| HP | 10th Generation | iLO5 | 2.63 or later
-
-.2+| Dell | 14th Generation | iDRAC 9 | v4.20.20.20 - v4.40.00.00 only
-
-| 13th Generation .2+| iDRAC 8 | v2.75.75.75 or later
+| Model | Management | Firmware versions
+| 10th Generation | iLO5 | 2.63 or later
 
 |====
 
 [NOTE]
 ====
-Red Hat does not test every combination of firmware, hardware, or other third-party components. For further information about third-party support, see link:https://access.redhat.com/third-party-software-support[Red Hat third-party support policy].
+For HP servers, Redfish virtual media is not supported on 9th generation systems running iLO4, because Ironic does not support iLO4 with virtual media.
```

Comment:
"Ironic does not support iLO4 with virtual media."  is factually false, we don't support it because iLO4 is not fully Redfish compliant, at least in its virtual media part. I'd suggest to omit the reason completely and only state that we support Redfish starting with iLO5.

Reply:
Thanks @dtantsur , so I think I can remove this note? As the table identifies compatibility with Redfish from iLO5

Reply:
I removed the note.